sdk/inc/ilttexturestring.h

```diff
@@ -20,6 +20,11 @@
 #	include "ilttexinterface.h"
 #endif
 
+#ifdef __LINUX
+#define LT_TEXTURE_STRING_TYPE char
+#else
+#define LT_TEXTURE_STRING_TYPE wchar_t
+#endif
```

Comment:
there really should only be one definition here
either the typedef or the define